ci/benchmark_parser.py

```diff
@@ -68,6 +76,18 @@ def recursive_parse(directory, walk_subdirs=False, name_suffix=""):
                 test_name_parts = list(filter(None, [test_name, stat_name, name_suffix]))
                 result_values.append({"value": value, "test": "_".join(test_name_parts)})
 
+                if stat_name == "mean" and throughput is True:
```

Comment:
just `throughput` it's a boolean :) or to be a bit more explicit rename it to "compute_throughput"